src/main/java/io/getstream/chat/java/models/App.java

```diff
@@ -763,6 +763,14 @@ public AppGetRateLimitsRequest endpoint(@NotNull String endpoint) {
     @JsonProperty("skip_devices")
     private Boolean skipDevices;
 
+    @Nullable
+    @JsonProperty("push_provider_name")
+    private String pushProviderName;
+
+    @Nullable
+    @JsonProperty("push_provider_type")
+    private String pushProviderType;
```

Comment:
Should be PushProviderType here once previous PR is merged

Reply:
✅